frontend/server/src/main/java/org/pytorch/serve/wlm/BatchAggregator.java

```diff
@@ -69,13 +71,12 @@ public BaseModelRequest getRequest(String threadName, WorkerState state)
      * @return - true: either a non-stream response or last stream response is sent - false: a
      *     stream response (not include the last stream) is sent
      */
-    public boolean sendResponse(ModelWorkerResponse message) {
-        boolean jobDone = true;
+    public void sendResponse(ModelWorkerResponse message) {
```

Comment:
The API change will break existing streaming response.

Reply:
see above